src/js/edu-benefits/actions/index.js

```diff
@@ -8,7 +8,7 @@ export const UPDATE_SUBMISSION_TIMESTAMP = 'UPDATE_SUBMISSION_TIMESTAMP';
 export const VETERAN_FIELD_UPDATE = 'VETERAN_FIELD_UPDATE';
 export const ENSURE_FIELDS_INITIALIZED = 'ENSURE_FIELDS_INITIALIZED';
 
-export function ensureFieldsInitialized(section) {
+export function ensureSectionInitialized(section) {
   return (dispatch, getState) => {
     return dispatch({
       type: ENSURE_FIELDS_INITIALIZED,
```

Comment:
Ah gotcha, that makes sense! Hadn't gotten to that part of the PR yet so I think that's fine.